oat-sa/extension-tao-backoffice
# Feature/ADF-910/e2e props with custom lists and trees

Related to https://oat-sa.atlassian.net/browse/ADF-910

### How to test
1) Checkout core https://github.com/oat-sa/tao-core/pull/3352
2) Checkout this extension/branch
3) Run `npm run cy:open` from `/tao/views`
4) Execute test: `taoBackOffice/views/cypress/tests/list.spec.js`
5) Observe results of the test: 4 success, 0 fields, 1 skip.

<img width="1904" alt="list" src="https://user-images.githubusercontent.com/2067027/150632341-c0aed3cb-c0a6-4a7e-85a4-4fcca0728ec6.png">

IMPORTANT:
PR using commands from https://github.com/oat-sa/tao-core/pull/3352.
This PR should be merged after https://github.com/oat-sa/tao-core/pull/3352.
